qiskit_ionq/ionq_client.py

```diff
@@ -97,6 +97,7 @@ def submit_job(self, job) -> dict:
             job.backend(),
             job._passed_args,
             job.extra_query_params,
+            job.extra_metadata,
```

Comment:
thoughts on a mode where this comes by default from the passsed in QuantumCircuit?

Reply:
got it thanks for the suggestion!